docs/src/main/asciidoc/web.adoc

```diff
@@ -0,0 +1,273 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Quarkus for the Web
```

Comment:
I think you should probably use frontend instead of web, or something similar. Web is very vague.

Reply:
This is not about frontend, though. It's for Web Applications. Front or back.